### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ElectroBoy404NotFound**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ Beta16 version image of ringOS
 - [ ] Book
 - [ ] Networking
 - [x] Own build tool
-- [ ] Real hardware support
+- [x] Real hardware support
```
Comment (Andrej123456789) [inline README.md:107]: ringOS can't run on real hardware after Beta11, so remove this as finished.

**Review by Andrej123456789 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ElectroBoy404NotFound) [reply README.md:107]: it can, i tried it

Comment (Andrej123456789) [reply README.md:107]: So you run Beta7 on real hardware.
Can I know your PC specification

Comment (Andrej123456789) [reply README.md:107]: Hello

Comment (ElectroBoy404NotFound) [reply README.md:107]: intel i7, 8gb ram, loaded from USB

Comment (Andrej123456789) [reply README.md:107]: Which version
Also, is that laptop

**Comment by ElectroBoy404NotFound** [comment]
Retried for pictures, does not work anymore

File: README.md (hunk 1 above)
Comment (ElectroBoy404NotFound) [reply README.md:107]: Yes it is a laptop

**Pull request closed without merging**